packages/media-sharing/src/MediaSharingModule.ts

```diff
@@ -22,13 +22,23 @@ export class MediaSharingModule implements Module {
 
     // Repositories
     dependencyManager.registerSingleton(MediaSharingRepository)
+  }
+
+  public async initialize(agentContext: AgentContext): Promise<void> {
+    const featureRegistry = agentContext.dependencyManager.resolve(DidCommFeatureRegistry)
+    const messageHandlerRegistry = agentContext.dependencyManager.resolve(DidCommMessageHandlerRegistry)
+    const mediaSharingService = agentContext.dependencyManager.resolve(MediaSharingService)
 
-    // Feature Registry
     featureRegistry.register(
-      new Protocol({
+      new DidCommProtocol({
         id: 'https://didcomm.org/media-sharing/1.0',
         roles: [MediaSharingRole.Sender, MediaSharingRole.Receiver],
       }),
     )
+
+    messageHandlerRegistry.registerMessageHandlers([
+      mediaSharingService.getShareHandler(),
+      mediaSharingService.getRequestHandler(),
+    ])
```

Comment:
I think we can follow here the same pattern as in other extension modules
```suggestion

    messageHandlerRegistry.registerMessageHandlers([
     new ShareMediaHandler(mediaSharingService),
      new RequestMediaHandler(mediaSharingService)
    ])
```
This way we eliminate the need of having those `getShareHandler` and `getRequestHandler`.

Reply:
Resolved in the commit [14273eb](https://github.com/2060-io/credo-ts-didcomm-ext/pull/137/commits/14273eba40ca9464972d490948d85df15e66cbf0)